src/cmd.rs

```diff
@@ -219,3 +473,203 @@ pub fn osd_crush_add(cluster_handle: rados_t, osd_id: u64, weight: f64, host: &s
     }
     Ok(())
 }
+
+// Luminous mgr commands below
```

Comment:
Would it be possible to find out the version of the ceph cluster through the socket, and pass around something more than a rados_t ? I'm considering that it should be doable to have something like:

```rust
// With Luminous cluster
let mut client =  CephClient::new(auth_data);
assert_eq!(client.version(), CephVersion::Luminous);
assert!(client.mgr_dump().is_ok());

//with Firefly cluster

let mut client =  CephClient::new(auth_data);
assert_eq!(client.version(), CephVersion::Firefly);
// This line shouldn't even try to send to cthe socket since we know it will fail
assert!(client.mgr_dump().is_err());
```